code/modules/mob/living/silicon/robot/robot.dm

```diff
@@ -672,7 +675,9 @@
 	if(stat != DEAD && !(IsUnconscious() || IsStun() || IsParalyzed() || low_power_mode)) //Not dead, not stunned.
 		if(!eye_lights)
 			eye_lights = new()
-		if(lamp_enabled)
+		if(last_flashed && last_flashed + 30 SECONDS >= world.time) //We want to make sure last_flashed isn't zero because otherwise roundstart borgs blink for 30 seconds
```

Comment:
```suggestion
		if(last_flashed + 30 SECONDS >= world.time) //We want to make sure last_flashed isn't zero because otherwise roundstart borgs blink for 30 seconds
```
if last_flashed is 0, then 30 SECONDS will always be less than world.time

Reply:
No it won't. World.time starts when the round starts.

I added this because borgs were spending the first 30 seconds of rounds flashing red in testing.